### PR TITLE
[UX] Absurdly aggressive method to ensure tooltips always work anywhere

### LINKED ISSUE #8883: Number field seems to offer help but there are no tooltips showing
## Bug Description
When testing the custom field creation, I noticed that on the number field type we have two 'buttons' that look like they should provide help or tooltips but they provide neither.

screenshot-local mautic3-2020 06 05-17_18_34

If the intent was to show some information, this needs to be added. If this isn't the intention then I think we should remove this as it is confusing.

| Q   | A
| --- | ---
| Mautic version | 3.0.0-beta-2
| PHP version | 7.2.31
| Browser | Chrome

### Steps to reproduce
1. Create a custom field using the field type of number
2. Noticed that the items have a button that looks like it should offer help but none is available

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #8883 

## Description

This PR really ensures that tooltips will work FOREVER, anywhere you place it, using MutationObserver.

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Follow https://github.com/mautic/mautic/issues/8883

### CODE REVIEW

**Review by Esthertests — CHANGES_REQUESTED** [review]
while testing i could see some html tags within the text, can these be removed 
image

**Comment by andersonjeccel** [comment]
@Esthertests Yep (in another PR, it's a bug not related to this one), thanks for testing!

**Review by Bastian2718 — APPROVED** [review]
Irritating Questionmark, shows nothing to me.

**Comment by andersonjeccel** [comment]
> [!WARNING]
> All Tier 1 PRs [need only 1 code review and 1 user testing](https://contribute.mautic.org/governance/code-governance#:~:text=Full%20testing%20and%20approval%20by%20at%20least%20one%20community%20member%20(could%20be%20the%20person%20doing%20the%20code%20review%2C%20but%20should%20not%20be%20from%20the%20same%20company/organisation%20as%20the%20person%20submitting%20the%20PR).) before able to merge. I'll update accordingly.

**Comment by andersonjeccel** [comment]
Testing passed based on Esther's test

File: app/bundles/CoreBundle/Assets/js/1.core.js [hunk 1]
```diff
@@ -67,6 +67,30 @@ mQuery(document).ready(function() {
         });
     }
 
+    // Absurdly aggressive method to ensure tooltips always work anywhere
```
Comment (LordRembo) [inline app/bundles/CoreBundle/Assets/js/1.core.js:70]: A bit of a similar issue as I described in #13948 : the code looks fine, but it shouldn't be in this location.
I'm fine with tooltips code existing in `core.js` if there's no better place for it, but you need to follow the style and formatting of the rest of the file:

- functions should be in the `Mautic` namespace (nesting starts around line 154, or you can prefix it like happens in other files)
- variables should be prefixed with `MauticVars` (as you can see above that)

**Review by LordRembo — CHANGES_REQUESTED** [review]
Added comment on the JS code about the formatting and location

File: app/bundles/CoreBundle/Assets/js/1.core.js (hunk 1 above)
Comment (andersonjeccel) [reply app/bundles/CoreBundle/Assets/js/1.core.js:70]: Oh, got it
It’ll be fixed asap

I’ll also take notes for future implementations 
Thanks for reviewing 🙏🏻

**Pull request closed without merging**